go-controller/pkg/util/kube.go

```diff
@@ -129,7 +129,10 @@ func GetClusterIPs(service *kapi.Service) []string {
 	if len(service.Spec.ClusterIPs) > 0 {
 		return service.Spec.ClusterIPs
 	}
-	return []string{service.Spec.ClusterIP}
+	if len(service.Spec.ClusterIP) > 0 || service.Spec.ClusterIP != kapi.ClusterIPNone {
```

Comment:
this is a fixup to the commit where you added this function right?

Reply:
yeah, Dan Winship noticed the bug I had

Reply:
this is wrong, is an AND not an OR